### PR TITLE
Update setup instructions

### PR DESCRIPTION
**Pull request by bugadani**

### CODE REVIEW

File: src/docs/getting-started/probe-setup.md [hunk 1]
```diff
@@ -116,3 +113,66 @@ this.
...
+
+#### Linux
+
+No additional drivers are required to use an ESP32 built-in debug interface on Linux systems.
+To ensure that users without root privileges can use the debug probe, it is
+recommended to configure udev as described in [udev rules](#udev-rules).
+
+#### Windows, Mac OS
+
+No driver installation required.
```
Comment (bugadani) [inline src/docs/getting-started/probe-setup.md:178]: In all honesty, I just forgot if anything is needed on Windows.

File: src/docs/getting-started/installation.md [hunk 2]
```diff
@@ -4,66 +4,40 @@ description: "How to install probe-rs."
 order: 20
 ---
 
+# Installing a precompiled binary (`cargo binstall`)
 
-# Prerequisites
+## Prerequisites
 
-To build probe-rs, you will need a working Rust toolchain. The easiest way is to follow the [rustup installation instructions](https://rustup.rs).
+First, set up `cargo-binstall`. You can find setup instructions [`here`](https://crates.io/crates/cargo-binstall).
```
Comment (Yatekii) [inline src/docs/getting-started/installation.md:11]: Don't we want to recommend installing via the bash script or brew? Think we should recommend the least effort.

File: src/docs/getting-started/installation.md [hunk 3]
```diff
@@ -4,66 +4,40 @@ description: "How to install probe-rs."
...
 
-Additionally, probe-rs depends on [libusb](https://libusb.info/) and optionally on [libftdi](https://www.intra2net.com/en/developer/libftdi/) libraries, which need to be installed to build probe-rs. Instructions for your OS are provided below.
+## Installation
 
-## Debian-based Linux
+`cargo binstall probe-rs`
```
Comment (Yatekii) [inline src/docs/getting-started/installation.md:15]: Does it not need the `--features cli` flag?

Comment (bugadani) [reply src/docs/getting-started/installation.md:15]: I don't think it does
image

File: src/docs/getting-started/installation.md (hunk 2 above)
Comment (bugadani) [reply src/docs/getting-started/installation.md]: Is probe-rs available via brew?

File: src/docs/getting-started/installation.md [hunk 4]
```diff
@@ -4,66 +4,40 @@ description: "How to install probe-rs."
...
-# dynamic linking 64-bit
-vcpkg install libftdi1:x64-windows libusb:x64-windows
-set VCPKGRS_DYNAMIC=1
-
-# static linking 64-bit
-vcpkg install libftdi1:x64-windows-static-md libusb:x64-windows-static-md
+sudo apt install -y pkg-config libudev-dev
 ```
 
-## macOS
```
Comment (Yatekii) [inline src/docs/getting-started/installation.md:49]: Maybe we should list macOS as nothing required still? Same for Windows, just to make sure folks don't think the section is missing.

File: src/docs/getting-started/installation.md (hunk 2 above)
Comment (Yatekii) [reply src/docs/getting-started/installation.md]: Yup :)

File: src/docs/getting-started/probe-setup.md (hunk 1 above)
Comment (Yatekii) [reply src/docs/getting-started/probe-setup.md:178]: =D

**Review by Yatekii — APPROVED** [review]
Amazing stuff, thank you!

File: src/docs/getting-started/installation.md (hunk 2 above)
Comment (bugadani) [reply src/docs/getting-started/installation.md]: really? fyi I couldn't find it on brew.sh so I removed the instr from here for now

**Pull request merged**